hack/Makefile

```
@@ -97,18 +97,18 @@ go-bindata:
 
 .PHONY: vault
 vault:
-	@if [ "$(VAULT)" != "$(VAULT_VERSION)" ]; then \
+ifneq ($(VAULT), $(VAULT_VERSION))
 		curl -o $(LOCALBIN)/vault.zip https://releases.hashicorp.com/vault/$(VAULT_VERSION)/vault_$(VAULT_VERSION)_$(OS_ARCH).zip; \
 		unzip -o $(LOCALBIN)/vault.zip -d $(LOCALBIN); \
 		rm $(LOCALBIN)/vault.zip; \
-		chmod a+x $(LOCALBIN)/vault;\
-	fi
+		chmod a+x $(LOCALBIN)/vault;
+endif
 
 .PHONY: oci-registry
 oci-registry:
-	@if [ "$(OCI_REGISTRY)" != "$(OCI_REGISTRY_VERSION)" ]; then \
-		go install -v github.com/distribution/distribution/v3/cmd/registry@v3.0.0-alpha.1; \
-	fi
+ifeq (,$(findstring $(OCI_REGISTRY_VERSION), $(OCI_REGISTRY)))
```

is the `,` correct? or should be removed?

```suggestion
ifeq ($(findstring $(OCI_REGISTRY_VERSION), $(OCI_REGISTRY)))
```

---

This is correct, although i agree, the syntax looks weird.
If `$OCI_REGISTRY` contains `$(OCI_REGISTRY_VERSION)`, [findstring](https://www.gnu.org/software/make/manual/make.html#index-findstring) returns the value of `$OCI_REGISTRY`. Otherwise, findstring returns an empty string.
What this statement this is to check that findstring returned an empty string (thus, we have to update the version).